src/write.ts

```diff
@@ -297,7 +300,12 @@ async function handleAlert(benchName: string, curSuite: Benchmark, prevSuite: Be
         // Note: alertThreshold is smaller than failThreshold. It was checked in config.ts
         const len = alerts.length;
         const threshold = floatStr(failThreshold);
-        const failures = alerts.filter((a) => a.ratio > failThreshold);
+
+        //Only consider it as failure if the benchmark performed better irrespective of the alert threshold
+        // For ex if there is a massive performance improvement between commits
+        // or the alert threshold and failure threshold is lower the failure might be triggered.
+        // This check ensures that the check is failure only if the performance is not better
```

Comment:
This comment is difficult to read. It starts with "Only consider it as failure if the benchmark performed better…" Isn't the purpose of this change not to fail if the benchmark performs better?